### PR TITLE
Add Host setting for receipt > bill to

### PR DESCRIPTION
**Pull request by Betree**
Resolve https://github.com/opencollective/opencollective/issues/5516
PDF service PR: https://github.com/opencollective/opencollective-pdf/pull/863

image

### CODE REVIEW

File: components/admin-panel/sections/InvoicesReceipts.js [hunk 1]
```diff
@@ -65,17 +80,15 @@ const InvoicesReceipts = ({ collective }) => {
   };
 
   const getInvoiceTemplatesObj = () => {
-    if (!alternativeReceiptTitle && !alternativeInfo) {
-      return {
-        templates: {
-          default: { title: receiptTitle || defaultReceiptTitlePlaceholder, info: info },
```
Comment (Betree) [inline components/admin-panel/sections/InvoicesReceipts.js:71]: Here I removed the `receiptTitle || defaultReceiptTitlePlaceholder`, we don't need to default on the placeholder, if the value is empty the PDF service will just use its default

File: components/admin-panel/sections/InvoicesReceipts.js [hunk 2]
```diff
@@ -57,6 +64,14 @@ const InvoicesReceipts = ({ collective }) => {
     get(data, 'editCollective.settings.invoice.templates.default.info') === info &&
     get(data, 'editCollective.settings.invoice.templates.alternative.info') === alternativeInfo;
 
+  // For Bill To
+  const getBillToOption = value => BILL_TO_OPTIONS.find(option => option.value === value) || BILL_TO_OPTIONS[0];
+  const [defaultBillTo, setDefaultBillTo] = React.useState(getInTemplate(collective, 'default', 'billTo'));
+  const [alternativeBillTo, setAlternativeBillTo] = React.useState(getInTemplate(collective, 'alternative', 'billTo'));
+  const billToIsSaved =
+    getInTemplate(data?.editCollective, 'default', 'billTo') === defaultBillTo &&
+    getInTemplate(data?.editCollective, 'default', 'alternative') === alternativeBillTo;
```
Comment (SudharakaP) [inline components/admin-panel/sections/InvoicesReceipts.js:73]: Shouldn't this be, 

```suggestion
    getInTemplate(data?.editCollective, 'alternative', 'billTo') === alternativeBillTo;
```

**Review by SudharakaP — APPROVED** [review]
Small comment below which I didn't understand, rest looks good to me. 👍🏽

**Review by Betree — COMMENTED** [review]
Updated the feature after a review with JF

**Review by SudharakaP — APPROVED** [review]

**Comment by SudharakaP** [comment]
> Updated the feature after a review with JF

Tested and seems all is fine. 💯

**Pull request merged**